lib/js/styles/Colors/Colors.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+	<div class="sb__presentation">
+		<div class="sb__row">
+			<ol class="sb__list">
+				<li v-for="(colors, colorCategory) in rawColors" :key="colorCategory" class="sb__legend">
+					{{ colorCategory }}
+				</li>
+			</ol>
+		</div>
+		<div v-for="(colors, colorCategory) in rawColors" :key="colorCategory" class="sb__row">
+			<h2>{{ colorCategory }}</h2>
+			<div v-for="color in colors" :key="color.id" class='sb__table'>
+				<div class="sb__tableId">{{ color.label }}</div>
+				<div class="sb__tableValue">{{ color.value }}</div>
+				<div class="sb__tableColor">
+					<span class="sb__color" :style="{ background: color.value }"></span>
```

Comment:
why do we need two separate DOM nodes? Can we move styles to parent node?

Reply:
I will move styles and remove class `sb_color` , but this span is needed to set some padding.